src/applications/veteran-id-card/config.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const config = {
+  messages: {
+    VIC002: (<p>We're sorry. We're having trouble finding your records in our system right now. Please go to <a href="https://access.va.gov/accessva/">AccessVA</a> to request a Veteran ID Card.</p>),
+    VIC003: "We're sorry. We can't proceed with your request for a Veteran ID card because we can't confirm your eligibility right now. Please call the Vets.gov Help Desk at 1-855-574-7286, TTY: 1-800-877-8339, Monday &#8211; Friday, 8:00 a.m. &#8211; 8:00 p.m. (ET).",
+    VIC010: "We're sorry. We can't proceed with your request for a Veteran ID card because we can't confirm your military history right now. Please try again in a few minutes. If it still doesn't work, please call the Vets.gov Help Desk at 1-855-574-7286, TTY: 1-800-877-8339, Monday &#8211; Friday, 8:00 a.m. &#8211; 8:00 p.m. (ET).",
+    VIC011: "We're sorry. We can't proceed with your request for a Veteran ID card because we can't confirm your military history right now. Please try again in a few minutes. If it still doesn't work, please call the Vets.gov Help Desk at 1-855-574-7286, TTY: 1-800-877-8339, Monday &#8211; Friday, 8:00 a.m. &#8211; 8:00 p.m. (ET).",
+    'default': "We're sorry, but something went wrong. Please try again in a few moments.",
+    VICV5: (<p>We're sorry. Something went wrong on our end. We can't process your request right now. Please go to <a href="https://access.va.gov/accessva/">AccessVA</a> to request a Veteran ID Card.</p>),
```

Comment:
`VICV5, VICV2, VICV4` are new error codes that correspond with the title 38 status from eMIS.